modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/fragments/group/src/customer-portal/fragments/information-banner/index.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+/* eslint-disable no-undef */
+
+const bannerElement = document.querySelectorAll('.cp-information-banner');
+const bannerFragment = fragmentElement.querySelector('.cp-information-banner');
+const closeButton = fragmentElement.querySelector('.close');
+
+const configurationDate = configuration.expirationDate;
+const currentDate = new Date();
+const {0: month, 1: day, 2: year} = configurationDate.split('/');
+const expirationDate = new Date(year, month - 1, day);
+const isExpired = expirationDate < currentDate;
+
+const bannerState = !sessionStorage.getItem('@liferayCP:showBanner')
+? true
+: false;
```

Comment:
i think the indentation got messed up on the `? true : false` during the refinement? i believe it was fine before. can you fix that?

Reply:
I ran formatSource and it adjusted, now everything is fine?